<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monster_Price</class>
 <widget class="QMainWindow" name="Monster_Price">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="Box_Money">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;----- PLEASE INSERT MONEY HERE</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>691</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="Cola_Price">
           <property name="font">
            <font>
             <family>Sylfaen</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>COLA PRICE : 10    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Sprite_Price">
           <property name="font">
            <font>
             <family>Sylfaen</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>SPRITE PRICE :8    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Dr_Price">
           <property name="font">
            <font>
             <family>Sylfaen</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>DR.PEPPER PRICE: 6      </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Sylfaen</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>MONSTER PRICE:4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="LB_Cola">
           <property name="font">
            <font>
             <family>Sylfaen</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>COLA COUNT: 5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LB_Sprite">
           <property name="font">
            <font>
             <family>Sylfaen</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>SPRITE COUNT: 5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LB_Dr">
           <property name="font">
            <font>
             <family>Sylfaen</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>DR.PEPPER COUNT: 5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LB_Monster">
           <property name="font">
            <font>
             <family>Sylfaen</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>MONSTER COUNT: 5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="BTN_Cola">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="text">
          <string>Cola</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BTN_Sprite">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="text">
          <string>Sprite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BTN_Dr">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="text">
          <string>Dr.Pepper</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BTN_Monster">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="text">
          <string>Monster</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="BTN_Setting">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SETTING</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
